Resources/public/scss/modules/sub-items-list/_table.view.item.scss

```diff
@@ -31,18 +33,25 @@
         }
 
         &--checkbox {
-            width: 5%;
+            position: absolute;
+            left: 0;
+            width: calculateRem(30px);
+            border-top: calculateRem(1px) solid $ez-ground-primary;
+            line-height: 0;
+            padding-top: calculateRem(16px);
+            margin-top: -0.5px;     // Fixes borders alignment on Safari and Edge
```

Comment:
This line deserves it's own comment. I had problem with alignment of borders on Edge and Safari:
<img width="56" alt="screen shot 2019-02-20 at 09 17 09" src="https://user-images.githubusercontent.com/10233057/53080953-1dec3b00-34fa-11e9-94e1-651e0326974d.png">


### Edge
_(pictures are big, for better view open in separate tab)_
![screen shot 2019-02-20 at 09 15 13](https://user-images.githubusercontent.com/10233057/53080784-bb933a80-34f9-11e9-90ab-596a30c814f8.png)

### Safari
<img width="1326" alt="screen shot 2019-02-20 at 09 16 25" src="https://user-images.githubusercontent.com/10233057/53080822-d1086480-34f9-11e9-8136-6118c8fc8611.png">

What is more interesting, when it comes to Safari problem was occurring on Retina display but not on FullHD display. My conclusion was that it boils down to rounding of position:
<img width="277" alt="screen shot 2019-02-20 at 09 16 25 copy" src="https://user-images.githubusercontent.com/10233057/53081496-46286980-34fb-11e9-858d-c73b68a62714.png">
It seems that Safari was rounding it to 43px and `margin-top: -0.5px;` fixes this problem. This is why I also don't want this value to be in `rem`.

Reply:
Maybe you should try adding a border on a row not on a cell itself, shouldn't you?

Reply:
But cells with checkboxes and action buttons are outside of the row, because they have position absolute.

Reply:
Then maybe it's not the cell that should be positioned absolutely but it's content? What do you think about creating a kind of a cell content wrapper with `div` and then position it absolutely?

Reply:
I don't think it would solve the problem, you would still need to have border but instead on wrapper div and not on the td.